### PR TITLE
show scope name in sidebar

### PR DESCRIPTION
**Pull request by odedre**
## Proposed Changes

- show scope name in sidebar tree
- fixed collapsing of nodes in sidebar

### CODE REVIEW

File: components/side-bar/component-tree/component-nodes/component-nodes.tsx [hunk 1]
```diff
@@ -10,14 +11,26 @@ import { getName } from '../utils/get-name';
...
+        <div className={classNames(indentClass, clickable, styles.namespace)} onClick={() => collapse(!collapsed)}>
+          <div className={styles.left}>
+            <Icon className={classNames(styles.arrow, collapsed && styles.collapsed)} of="fat-arrow-down" />
+            <Icon className={styles.arrow} of="collection-full" />
+            <span className={styles.name}>{displayName}</span>
+          </div>
+        </div>
+      )}
+      <AnimateHeight height={collapsed ? 0 : 'auto'}>
+        <div style={indentStyle(depth + 2)} className={classNames(styles.componentTree)}>
```
Comment (KutnerUri) [inline components/side-bar/component-tree/component-nodes/component-nodes.tsx:29]: please leave a comment about the extra depth

File: components/side-bar/component-tree/component-view/component-view.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export function ComponentView(props: ComponentViewProps<PayloadType>) {
 
   return (
     <NavLink
-      href={`/${node.id}`}
+      href={`/${component.id.fullName}`}
```
Comment (KutnerUri) [inline components/side-bar/component-tree/component-view/component-view.tsx:37]: it's the same thing, no?

**Comment by KutnerUri** [comment]
cool feature, resolve the comments and merge

**Review by KutnerUri — APPROVED** [review]
cool feature, resolve the comments and merge

File: components/side-bar/component-tree/component-view/component-view.tsx (hunk 2 above)
Comment (odedre) [reply components/side-bar/component-tree/component-view/component-view.tsx:37]: no. the node id now gets the scope in it as well

Comment (KutnerUri) [reply components/side-bar/component-tree/component-view/component-view.tsx:37]: I see

**Pull request merged**